release/e2e.py

```diff
@@ -1317,11 +1380,35 @@ def _run(logger):
                 _process_finished_command(
                     session_controller=session_controller, scd_id=scd_id)
             else:
+                timeout_type = ""
+                runtime = None
+                if isinstance(e, CommandTimeoutError):
+                    timeout_type = "timeout"
+                    runtime = 0
```

Comment:
For app failure, we report runtime = 0 and for infra failure we report None. This was the @ericl's request to plot runtime easily on the dashboard.